symbolic-debuginfo/src/breakpad.rs

```diff
@@ -1525,12 +1828,12 @@ mod tests {
         let record = BreakpadLineRecord::parse(string)?;
 
         insta::assert_debug_snapshot!(record, @r###"
-       ⋮BreakpadLineRecord {
-       ⋮    address: 14744848,
-       ⋮    size: 5,
-       ⋮    line: 4294966920,
-       ⋮    file_id: 2225,
-       ⋮}
+        BreakpadLineRecord {
+            address: 14744848,
+            size: 5,
+            line: 0,
```

Comment:
@jan-auer mentioned this to me earlier, as this is papering over a bug in other tools creating invalid sym files. In clamping to `0` the correct approach in this case?

Reply:
I think it's the most sensible option, since 0 is used to indicate that we don't know the line number. If a negative number comes in we really have nothing to go on.